src/status_im/contexts/wallet/send/events.cljs

```diff
@@ -182,32 +147,38 @@
 
 (rf/reg-event-fx
  :wallet/set-token-to-send
- (fn [{:keys [db]} [{:keys [token-symbol token stack-id start-flow? owners]} entry-point]]
+ (fn [{:keys [db]}
+      [{:keys [token-symbol token network stack-id start-flow? owners] :as params} entry-point]]
    ;; `token` is a map extracted from the sender, but in the wallet home page we don't know the
    ;; sender yet, so we only provide the `token-symbol`, later in
    ;; `:wallet/select-from-account` the `token` key will be set.
-   (let [{:keys [networks]}     token
-         receiver-networks      (get-in db [:wallet :ui :send :receiver-networks])
-         token-networks-ids     (map :chain-id networks)
-         unsupported-token?     (not-any? (set receiver-networks) token-networks-ids)
+   (let [{:keys [wallet]}       db
          unique-owner           (when (= (count owners) 1)
                                   (first owners))
          unique-owner-tokens    (get-in db [:wallet :accounts unique-owner :tokens])
          token-data             (or token
                                     (when (and token-symbol unique-owner)
                                       (some #(when (= (:symbol %) token-symbol) %)
                                             unique-owner-tokens)))
+         view-id                (:view-id db)
+         root-screen?           (or (= view-id :wallet-stack) (nil? view-id))
+         multi-account-balance? (-> (utils/get-accounts-with-token-balance (:accounts wallet)
+                                                                           token)
+                                    (count)
+                                    (> 1))
          test-networks-enabled? (get-in db [:profile/profile :test-networks-enabled?])
          network-details        (-> (get-in db
                                             [:wallet :networks (if test-networks-enabled? :test :prod)])
-                                    (network-utils/sorted-networks-with-details))]
+                                    (network-utils/sorted-networks-with-details))
+         next-screen            (:screen-id (wizard-utils/wizard-find-next-screen db
```

Comment:
This looks very suspicious :)
Could you please explain what you are trying to achieve by checking the next screen?

We are inside re-frame event and it should operate with the data stored inside db and make decisions based on that data. Maybe some flags can be passed as paramaters. But I'm pretty sure this code shouldn't know nothing about such UI concept as wizard :)

Reply:
In the current logic, basically we go to next screen only if a network is previously selected. As the network selector is a bottom sheet and not a screen, we do this check. If the next screen is `:screen/wallet.select-from` but don't have a network selected, we show the network selection drawer. Maybe we can improve this logic. Any suggestions?

Reply:
@briansztamfater, I dived a bit more in code :)

Later in `:fx` section we do the check:
```clojure
(if (or (= next-screen :screen/wallet.select-from) (some? network))
...
```

And `next-screen` is obtained by looking into wizard and finding next screen of :wallet-send-flow.
But wallet-send-flow doesn't have `:screen/wallet.select-from` screen at all, you removed it in this pr :) So the check above is always false and we can leave only `(some? network)` part.

Is it correct or am I missing something?

Reply:
Even if the wizard step was removed accidentally, we still shouldn't make any decisions based on wizard state. Instead we need to make decisions based on state of rf-db. 
In the send flow we have different parameters that need to be specified - `token`, `network`, `from-address`, `to-address. So in order to decide on next screen/bottomsheet we can check these parameters instead of wizard state.

Reply:
Sorry about the confusion, you are totally right, and great catch. We don't need this code, this must have been some leftover from my initial implementation and enver removed it. Just checking for `(some? network)` should be fine. Will remove this part!

Reply:
Done